### PR TITLE
Add empty service

### PR DESCRIPTION
**Pull request by TheDioBrando**

### CODE REVIEW

**Comment by TIove** [comment]
Kindly add GH workflows. As an e.g. https://github.com/LT-Students/DigitalOffice-UserService/blob/develop/.github/workflows/main.yml

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,188 @@
+# Rules in this file were initially inferred by Visual Studio IntelliCode from the D:\DigitalOffice\User codebase based on best match to current usage at 9/1/2021
```
Comment (TIove) [inline .editorconfig]: It's better to use our defaults editorconfigs instead, e.g. from [UserService](https://github.com/LT-Students/DigitalOffice-UserService/blob/develop/.editorconfig). 'Cause every organization has a little different code-style.

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,14 @@
+FROM mcr.microsoft.com/dotnet/sdk:6.0-bullseye-slim AS build
```
Comment (TIove) [inline Dockerfile]: sdk:7.0 everywhere in this file. You can find needed version [here](https://hub.docker.com/_/microsoft-dotnet-aspnet)

File: src/EventService/EventService.csproj [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+    <ProjectReference Include="..\EventService.Broker\EventService.Broker.csproj" />   
+    <ProjectReference Include="..\EventService.Business\EventService.Business.csproj" />   
+    <ProjectReference Include="..\EventService.Data.Provider.MsSql.Ef\EventService.Data.Provider.MsSql.Ef.csproj" />
+    <ProjectReference Include="..\EventService.Validation\EventService.Validation.csproj" />
+
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="AspNetCore.HealthChecks.SqlServer" Version="6.0.2" />
+    <PackageReference Include="AspNetCore.HealthChecks.UI.Client" Version="6.0.5" />
+    <PackageReference Include="Microsoft.AspNetCore.Mvc.NewtonsoftJson" Version="6.0.8" />
```
Comment (TIove) [inline src/EventService/EventService.csproj]: Update nuget versions:
NewtonsoftJson 7.0.2
Serilog.AspNetCore 6.1.0
Serilog.Settings.Configuration 3.4.0
Serilog.Sinks.Seq 5.2.2

File: src/EventService/Startup.cs [hunk 4]
```diff
@@ -0,0 +1,158 @@
...
+        options.UseSqlServer(dbConnStr);
+      });
+
+      services.AddHttpContextAccessor();
+
+      services.AddHealthChecks()
+        .AddRabbitMqCheck()
+        .AddSqlServer(dbConnStr);
+
+
```
Comment (TIove) [inline src/EventService/Startup.cs]: Delete extra spaces

**Review by liza459 — APPROVED** [review]

**Review by Draktolli — APPROVED** [review]

File: Common.Configurations.targets [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+    <Optimize>true</Optimize>
+    <DefineConstants>TRACE;RELEASE</DefineConstants>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="$(AssemblyName.Contains('.UnitTests'))">
+    <IsPackable>false</IsPackable>
+  </PropertyGroup>
+
+  <ItemGroup Condition="($(AssemblyName.Contains('.Provider')) Or $(AssemblyName.Contains('.Models.Db'))) And $(AssemblyName.Contains('.UnitTests')) != 'true'">
+  </ItemGroup>
```
Comment (vorobevamv) [inline Common.Configurations.targets]: lines 28-29 - what for?

File: src/EventService/EventService.csproj [hunk 6]
```diff
@@ -0,0 +1,25 @@
+﻿<Project Sdk="Microsoft.NET.Sdk.Web">
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+  <ItemGroup>
+    <ProjectReference Include="..\EventService.Broker\EventService.Broker.csproj" />   
+    <ProjectReference Include="..\EventService.Business\EventService.Business.csproj" />   
+    <ProjectReference Include="..\EventService.Data.Provider.MsSql.Ef\EventService.Data.Provider.MsSql.Ef.csproj" />
```
Comment (vorobevamv) [inline src/EventService/EventService.csproj]: don't need it

File: src/EventService/Startup.cs (hunk 4 above)
Comment (TheDioBrando) [reply src/EventService/Startup.cs]: Done

File: src/EventService.Validation/EventService.Validation.csproj [hunk 7]
```diff
@@ -0,0 +1,16 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(ProjectDir)..\..\Common.Configurations.targets" />
+  
+  <ItemGroup>
+    <ProjectReference Include="..\EventService.Broker\EventService.Broker.csproj" />
```
Comment (vorobevamv) [inline src/EventService.Validation/EventService.Validation.csproj]: most likely don't need it

File: src/EventService/appsettings.json [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+        "Args": {
+          "serverUrl": "http://localhost:5341",
+          "apiKey": "PRLXBdmLmIGBbDsD3AkT"
+        }
+      }
+    ]
+  },
+  "ConnectionStrings": {
+    "Redis": "localhost:6379"
+  },
```
Comment (vorobevamv) [inline src/EventService/appsettings.json]: don't need redis for now, remove lines 25-27

File: src/EventService/appsettings.Development.json [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  "ConnectionStrings": {
+    "SQLConnectionString": "Server=localhost\\sqlexpress;Database=EventDB;Trusted_Connection=True"
+  },
+  "RabbitMQ": {
+    "Host": "localhost",
+    "Username": "guest",
+    "Password": "guest"
+  },
+  "Redis": {
+    "CacheLiveInMinutes": 20
```
Comment (vorobevamv) [inline src/EventService/appsettings.Development.json]: remove redis

File: src/EventService/appsettings.Production.json [hunk 10]
```diff
@@ -0,0 +1,13 @@
...
+  "Serilog": {
+  },
+  "ConnectionStrings": {
+    "SQLConnectionString": "Server=127.0.0.1,20340;User=sa;Password=Event_1234;Database=EventDB;"
+  },
+  "RabbitMQ": {
+    "Host": "lt_do_rabbitmq"
+  },
+  "Redis": {
+    "CacheLiveInMinutes": 10
```
Comment (vorobevamv) [inline src/EventService/appsettings.Production.json]: remove redis

File: src/EventService/Startup.cs [hunk 11]
```diff
@@ -0,0 +1,158 @@
...
+namespace EventService
+{
+  public class Startup : BaseApiInfo
+  {
+    public const string CorsPolicyName = "LtDoCorsPolicy";
+    public IConfiguration Configuration { get; }
+
+    private readonly BaseServiceInfoConfig _serviceInfoConfig;
+    private readonly RabbitMqConfig _rabbitMqConfig;
```
Comment (vorobevamv) [inline src/EventService/Startup.cs]: move private fields: private first, then public
```
    private readonly BaseServiceInfoConfig _serviceInfoConfig;
    private readonly RabbitMqConfig _rabbitMqConfig;

    public const string CorsPolicyName = "LtDoCorsPolicy";
    public IConfiguration Configuration { get; }
```

File: src/EventService/Startup.cs [hunk 12]
```diff
@@ -0,0 +1,158 @@
...
+
+    public Startup(IConfiguration configuration)
```
Comment (vorobevamv) [inline src/EventService/Startup.cs:57]: please, fix the order:
private methods
constructor
public methods

File: src/EventService/Startup.cs [hunk 13]
```diff
@@ -0,0 +1,158 @@
...
+        .GetSection(BaseRabbitMqConfig.SectionName)
+        .Get<RabbitMqConfig>();
+
+      Version = "1.0.0.0";
+      Description = "EventService is an API that intended to work with events.";
+      StartTime = DateTime.UtcNow;
+      ApiName = $"LT Digital Office - {_serviceInfoConfig.Name}";
+    }
+
+    // This method gets called by the runtime. Use this method to add services to the container.
```
Comment (vorobevamv) [inline src/EventService/Startup.cs]: remove comment

File: src/EventService/Startup.cs [hunk 14]
```diff
@@ -0,0 +1,158 @@
...
+
+      services.AddHttpContextAccessor();
+
+      services.AddHealthChecks()
+        .AddRabbitMqCheck()
+        .AddSqlServer(dbConnStr);
+
+
+
+      services.AddControllers();
```
Comment (vorobevamv) [inline src/EventService/Startup.cs]: you have it on line 94, remove line 84

File: .editorconfig (hunk 1 above)
Comment (TheDioBrando) [reply .editorconfig]: Copied .editorconfig from PositionService https://github.com/LT-Students/DigitalOffice-PositionService/blob/develop/.editorconfig

File: Common.Configurations.targets (hunk 5 above)
Comment (TheDioBrando) [reply Common.Configurations.targets]: I dont know. Removed this lines

File: Dockerfile (hunk 2 above)
Comment (TheDioBrando) [reply Dockerfile]: Changed the version to 7.0

File: src/EventService.Validation/EventService.Validation.csproj (hunk 7 above)
Comment (TheDioBrando) [reply src/EventService.Validation/EventService.Validation.csproj]: Removed some dependencies

File: src/EventService/EventService.csproj (hunk 6 above)
Comment (TheDioBrando) [reply src/EventService/EventService.csproj]: Removed Broker, Business, Validation references

File: src/EventService/Startup.cs (hunk 11 above)
Comment (TheDioBrando) [reply src/EventService/Startup.cs]: Replaced fields

File: src/EventService/Startup.cs (hunk 12 above)
Comment (TheDioBrando) [reply src/EventService/Startup.cs:57]: Replaced methods

File: src/EventService/Startup.cs (hunk 13 above)
Comment (TheDioBrando) [reply src/EventService/Startup.cs]: Removed

File: src/EventService/Startup.cs (hunk 14 above)
Comment (TheDioBrando) [reply src/EventService/Startup.cs]: Done

File: src/EventService/appsettings.Development.json (hunk 9 above)
Comment (TheDioBrando) [reply src/EventService/appsettings.Development.json]: Removed

File: src/EventService/appsettings.Production.json (hunk 10 above)
Comment (TheDioBrando) [reply src/EventService/appsettings.Production.json]: Removed

File: src/EventService/appsettings.json (hunk 8 above)
Comment (TheDioBrando) [reply src/EventService/appsettings.json]: Removed

File: Dockerfile (hunk 2 above)
Comment (TIove) [reply Dockerfile]: You didn't change 1 line

File: src/EventService/Startup.cs [hunk 15]
```diff
@@ -0,0 +1,153 @@
...
+
+      services.AddControllers()
+        .AddJsonOptions(options =>
+       {
+         options.JsonSerializerOptions.Converters.Add(new JsonStringEnumConverter());
+       })
+        .AddNewtonsoftJson();
+    }
+
+    // This method gets called by the runtime. Use this method to configure the HTTP request pipeline.
```
Comment (TIove) [inline src/EventService/Startup.cs]: Remove this comment too

File: src/EventService/appsettings.Development.json [hunk 16]
```diff
@@ -0,0 +1,19 @@
...
+  },
+  "Kestrel": {
+    "EndPoints": {
+      "Http": {
+        "Url": "http://[:]:9868"
+      }
+    }
+  },
+  "ConnectionStrings": {
+    "SQLConnectionString": "Server=Uvarov;Database=EventDB;Trusted_Connection=True;TrustServerCertificate=True"
```
Comment (TIove) [inline src/EventService/appsettings.Development.json]: Change to default

Comment (TheDioBrando) [reply src/EventService/appsettings.Development.json]: Changed

File: src/EventService/Startup.cs (hunk 15 above)
Comment (TheDioBrando) [reply src/EventService/Startup.cs]: Removed

**Review by Segway-svg — APPROVED** [review]

**Review by TIove — APPROVED** [review]

File: Dockerfile (hunk 2 above)
Comment (TheDioBrando) [reply Dockerfile]: Changed version at first line

**Review by vorobevamv — APPROVED** [review]

**Pull request merged**